pkg/helm/helm_template.go

```diff
@@ -532,7 +532,11 @@ func (h *HelmTemplate) deleteResourcesBySelector(ns string, kinds []string, sele
 		}
 		output, err := h.runKubectlWithOutput(args...)
 		if err != nil {
-			errList = append(errList, err)
+			if warn {
```

Comment:
How dod you know that this error is caused by permission and not by any other error? Would you not have to inspect the error? 

I am not a big fan of this indirect coding. Would it not make more sense to have a function which checks whether you have cluster-wide permissions? I can see this being valuable for other use cases as well.